src/main/scala/com/gu/zuora/ZuoraService.scala

```diff
@@ -70,3 +71,36 @@ class ZuoraService(config: ZuoraConfig, client: FutureHttpClient, baseUrl: Optio
     decode[List[ZuoraErrorResponse]](responseBody).map(_.head)
 
 }
+
+object GetAccountForIdentity {
+
+  case class ZuoraAccountNumber(value: String)
+
+  case class DomainAccount(accountNumber: ZuoraAccountNumber, maybeCreatedSessionId: Option[SessionId])
+
+  object DomainAccount {
+
+    def fromWireAccount(accountRecord: AccountRecord): DomainAccount =
+      DomainAccount(
+        ZuoraAccountNumber(accountRecord.AccountNumber),
+        Some(accountRecord.CreatedSessionId__c).filter(_.length > 0).map(SessionId.apply)
+      )
+
+  }
+
+}
+
+object GetSubscription {
+
+  case class ZuoraIsActive(value: Boolean) extends AnyVal
+
+  case class DomainSubscription(accountNumber: ZuoraAccountNumber, isActive: ZuoraIsActive, ratePlans: List[RatePlan])
+  object DomainSubscription {
+    def fromWireSubscription(subscription: Subscription): DomainSubscription =
+      DomainSubscription(
+        ZuoraAccountNumber(subscription.accountNumber),
+        ZuoraIsActive(subscription.status == "Active"),
+        subscription.ratePlans //.map(DomainRatePlan.fromWire)
```

Comment:
no need to make RatePlan into a domain model at this stage because I didn't really change that area or notice any specific benefits from adding the type safety now.  Happy to reconsider otherwise I'll delete the comment.